components/n-ui/tracking/ft/utils/abTestHelpers.js

```diff
@@ -0,0 +1,17 @@
+
+const getTeaserTestContext = function (doc) {
+
+	const teasersUnderTest = [].slice.call(doc.querySelectorAll('[data-trackable-context-teaser-variant]'));
```

Comment:
You can use `Array.from()` here (it should be polyfilled by default for older browsers)